cpu/lm4f120/include/periph_cpu.h

```diff
@@ -115,6 +115,27 @@ typedef enum {
     ADC_RES_16BIT = 0xd00,            /**< not supported by hardware */
 } adc_res_t;
 
+
+/**
+ * @name SPI configuration
+ * @{
+ */
+#define SPI_NUMOF 4
```

Comment:
Ok, that's the only reason I've been able to find after reading the comment. I'll do a cleanup of all macros !